lang/python/python.talon

```diff
@@ -107,4 +107,4 @@ state past: "pass"
 #^pro static funky <user.text>$: user.code_protected_static_function(text)
 #^pub static funky <user.text>$: user.code_public_static_function(text)
 raise {user.python_exception}: user.insert_cursor("raise {python_exception}([|])")
-
+dock string: user.insert_cursor("\"\"\"[|]\"\"\"")
```

Comment:
I think this is a good idea, though I think it's worth taking a moment to make it language-neutral via code.py. 

Then we could move it into programming.talon and use it for all languages
`dock string: user.code_document_string()`

then python here would implement it
```
actions(user.code_document_string):
    user.insert_cursor("\"\"\"[|]\"\"\"")
```

as this does vary per language.
Thoughts?

Reply:
Perhaps it would need a different name as doc *strings* are a bit of a python thing, other languages typically use comments like
```
/* ... */
```